public/index.html

```diff
@@ -0,0 +1,45 @@
+ <!DOCTYPE html>
+ <html lang="en">
+ <head>
+     <meta charset="UTF-8">
+     <title>Document</title>
+     <link rel="stylesheet" type="text/css" href="styles/game_of_life.css">
+     
+ </head>
+<body id="body">
+    <form>
+        <label for="fname">Columns</label><br>
+        <input type="text" id="columns" name="wid" value=""><br>
+        <label for="lname">Rows</label><br>
+        <input type="text" id="rows" name="hei" value=""><br><br>
+        <button type="button" id="submit-button">Submit</button>
+        <button type="button" id="simulate-button">Simulate</button>
+        <button type="button" id="clear-button">Clear</button>
+        <label for="animate" class="animateLabel">Animate
```

Comment:
You can make animate work by using timeouts. See https://www.w3schools.com/jsref/met_win_settimeout.asp